src/main/java/bio/terra/cli/service/FeatureService.java

```diff
@@ -9,34 +9,21 @@
 import com.flagsmith.models.Flags;
 import java.util.HashMap;
 import java.util.Map;
-import java.util.Optional;
 import javax.annotation.Nullable;
 import org.apache.http.util.TextUtils;
 import org.slf4j.Logger;
 
+// TODO(BENCH-1050) use common code + exception from TCL
+
 public class FeatureService {
   private static final Logger LOGGER = getLogger(FeatureService.class);
-  private final FlagsmithClient flagsmith;
 
   // list of features
-  public enum Features {
-    AWS_ENABLED("vwb__aws_enabled"),
-    GCP_ENABLED("vwb__gcp_enabled"),
-
-    // CLI specific
-    CLI_AUTH0_TOKEN_REFRESH_ENABLED("vwb__cli_token_refresh_enabled"),
-    CLI_DATAPROC_ENABLED("vwb__cli_dataproc_enabled");
-
-    private final String featureName;
-
-    Features(String featureName) {
-      this.featureName = featureName;
-    }
-
-    public String toString() {
-      return featureName;
-    }
-  }
+  public static final String AWS_ENABLED = "vwb__aws_enabled";
+  public static final String GCP_ENABLED = "vwb__gcp_enabled";
```

Comment:
why do we have gcp enabled? gcp is always enabled

Reply:
No this is not needed right now. removed. It was left over from a previous change